uniffi/tests/ui/version_mismatch.stderr

```diff
@@ -0,0 +1,7 @@
+error[E0080]: evaluation of constant value failed
+ --> $DIR/version_mismatch.rs:2:1
+  |
+2 | uniffi::assert_compatible_version!("0.0.1"); // An error message would go here.
+  | ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ attempt to compute `0_usize - 1_usize`, which would overflow
+  |
+  = note: this error originates in a macro (in Nightly builds, run with -Z macro-backtrace for more info)
```

Comment:
This is the error experience that consumers would get if they don't have aligned versions. It's...not awesome, but I reckon it would be good enough, and is certainly less confusing than getting a false-positive compilation error like I got over in https://github.com/mozilla/application-services/pull/3904#issuecomment-790224056